scripts/checks/coverage.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+export COVERAGE=true
+export FOUNDRY_FUZZ_RUNS=10
+
+# Hardhat coverage
+npx hardhat coverage
```

Comment:
npx should not needed here (see `scripts/prepare-docs.sh`)

```suggestion
hardhat coverage
```